backend/core/src/services/application-formatting-metadata.ts

```diff
@@ -2,10 +2,9 @@ import {
   booleanFormatter,
   defaultFormatter,
   dobFormatter,
-  FormattingMetadataAggregate,
-  FormattingMetadataAggregateFactory,
+  FormattingMetadataAggregateFactory, joinArrayFormatter,
```

Comment:
Pedantic, but this dependency should be on its own line. The Prettier ESLint config should have taken care of that automatically, are you running pre-commit hooks?

Reply:
Indeed, the linter was failing in CI too. I went ahead and fixed it so we can merge, but @pbn4, if you can see if you can get things working in your local environment (and feel free to make a PR with config changes, if needed), that would be appreciated.